enforcer-rules/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.cloud.tools.opensource</groupId>
+    <artifactId>cloud-tools-opensource-parent</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>linkage-checker</artifactId>
+
+  <name>Linkage Checker Enforcer Rule</name>
+  <url>https://github.com/GoogleCloudPlatform/cloud-opensource-java/enforcer-rules</url>
```

Comment:
an scm section could be useful:

https://maven.apache.org/pom.html#SCM

Reply:
Added scm